### PR TITLE
Mongo integration

### PR DESCRIPTION
**Pull request by limonana**
create InitDB application to fill mongo with initial data
implement  getMovieById to query the db

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
```
Comment (yarindeoh) [inline .vscode/launch.json]: is this part of the IDE conf ? 
if so, we can insert it to gitignore

Comment (limonana) [reply .vscode/launch.json]: yes. I forgot about this option

**Comment by limonana** [comment]
@yarindeoh sorry but I commented out unneeded CRUD operation as we agree they are not needed right now and I didn't want to  implement them if not needed.

**Comment by limonana** [comment]
something I didn't succeed figure out is what are the best practices connecting to mongo. 
Should I save the Db object returned from dbServer.db('rate-movies') and use it always or is it OK the way it is done now. because I didn't succeed in figuring this out, I left the code as it because it is simpler.

**Pull request merged**